pkg/control/sidecarcontrol/util.go

```diff
@@ -54,6 +54,8 @@ const (
 	// SidecarsetInplaceUpdateStateKey records the state of inplace-update.
 	// The value of annotation is SidecarsetInplaceUpdateStateKey.
 	SidecarsetInplaceUpdateStateKey string = "kruise.io/sidecarset-inplace-update-state"
+
+	SidecarSetRevisionIDLabel = "apps.kruise.io/sidecarset-revision-id"
```

Comment:
1. It should be defined in `apis/apps/v1alpha1/sidecarset_types.go`, since it is an API for users.
2. How about `apps.kruise.io/sidecarset-custom-version`?